src/main/java/first/dto/StudDto.java

```diff
@@ -0,0 +1,112 @@
+package first.dto;
+
+public interface StudDto {
+
+    String FORM = "<html>\n" +
+            " <head>\n" +
+            " </head>\n" +
+            " <meta http-equiv=\"Content-Type\"  content=\"text/html; charset=utf-8\">\n" +
+            " <body> \n" +
+            " <form action=\"/viewstud\" method=\"get\" >\n" +
+            "  <p>Имя:<br>\n" +
+            "   <input type=\"text\" name=\"name\" size=\"40\">\n" +
+            "  </p>\n" +
+            "  <p>Фамилия:<br>\n" +
+            "   <input type=\"text\" name=\"surname\" size=\"40\">\n" +
+            "  </p>\n" +
+            "  <p>Возраст:<br>\n" +
+            "   <input type=\"number\" name=\"age\" size=\"2\">\n" +
+            "  </p>\n" +
+            "  <p>Оценка:<br>\n" +
+            "   <input type=\"number\" name=\"mark\" size=\"2\">\n" +
+            "  </p>\n" +
+            "  <p>Курс:<br>\n" +
+            "   <input type=\"number\" name=\"course\" size=\"1\">\n" +
+            "  </p>\n" +
+            "   <p><input type=\"submit\" name=\"button\" value=\" Добавить \"></p>\n" +
+            "  </form>\n" +
+            " </body>\n" +
+            "</html>";
+
+    String str = " <head>\n" +
+            " </head>\n" +
+            " <meta http-equiv=\"Content-Type\"  content=\"text/html; charset=utf-8\">\n" +
+            " <body> \n" +
+            " <form action=\"/form\" method=\"get\" >\n" +
+            " <p><input type=\"submit\" name=\"button\" value=\" Добавить студента \"></p>\n" +
+            "  </form>\n" +
+            " <form action=\"/show\" method=\"get\" >\n" +
+            " <p><input type=\"submit\" name=\"button\" value=\" Посмотреть список \"></p>\n" +
+            "  </form>\n" +
+            " </body>\n" +
+            "</html>";
+
+    String strEdit = " <head>\n" +
```

Comment:
What a difference between str and strEdit and str in StudentControllerInter? As I see - it is a duplicate.